### DISCUSSION TITLE
ЛР 6 для ИВТ. Неправильное сравнение с mcause в программе из примера

### DISCUSSION THREAD

**Discussion by BROsandr** [MPSU/exAPS#46] [Ошибки]
![image](https://user-images.githubusercontent.com/53278658/206222256-daded977-acbb-49ea-8b9f-7e312a761cf1.png)
mcause сравнивается с 5ю, но сам mcause формируется с 1 в старшем бите.
![image](https://user-images.githubusercontent.com/53278658/206222715-2b1e0953-5b39-4e8e-83cf-fb0a7e1cc32d.png)
Поэтому программа прерывания не обнаруживает

**Comment by gr33nka** [comment]
Верно, спасибо, вместо этого должно быть:
``` assembly risc-v
li t2, 0x80000005        # t2 = 0x80000005 (код одного из прерываний)
```
Думаю @HepoH3 отразит это как-то в журнале

**Reply by HepoH3** [reply]
Да, докинул балл, спасибо за фидбек!
(проверь что в ориоксе балл стоит, ставил по памяти).
